docs/source/markdown/options/retry-delay.md

```diff
@@ -1,7 +1,8 @@
 ####> This option file is used in:
-####>   podman build, farm build, pull
+####>   podman build, create, farm build, pull, push, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
 #### **--retry-delay**=*duration*
 
-Duration of delay between retry attempts in case of failure when performing pull of images from registry. Default is **2s**.
+Duration of delay between retry attempts when pulling or pushing images between
+registry and local storage in case of failure. Default is to start at two seconds and then exponentially backoff. When this value is set it the delay is used, and no exponential back off occurs.
```

Comment:
```suggestion
the registry and local storage in case of failure. The default is to start at two seconds and then exponentially back off. The delay is used when this value is set, and no exponential back off occurs.
```